crates/sui/src/keytool.rs

```diff
@@ -135,6 +135,10 @@ impl KeyToolCommand {
                     Ok(keypair) => {
                         println!("Public Key: {}", keypair.public().encode_base64());
                         println!("Flag: {}", keypair.public().flag());
+                        if let PublicKey::Ed25519(public_key) = keypair.public() {
```

Comment:
Do we enforce validators using different anemo keys? or in theory they can "on purpose" share one?

Reply:
we only support ed25519 for anemo today

Reply:
This is also true of any of our keys that i'm aware of

Reply:
opened an issue here: https://github.com/MystenLabs/sui/issues/9351